Warning explanation about the connection username (#484)
akeneo/pim-helpcenter: content/md/connectivity/manage-your-connections.md

Use these credentials to authenticate the API calls in the connector that matches your connection. For this step, take a look at this [guide](https://api.akeneo.com/getting-started/connect-the-pim-4x/step-3.html).

## Why should you use the connection username?

You may have noticed that nothing prevents you from using another username than the one generated for you when your connection is created. It's totally possible but it's, in fact, not recommended at all!

> Instead, we highly **recommend using the connection username** to authenticate all the API calls made by the connector linked to your connection.

Reviewer: ```suggestion
Instead, we highly **recommend using the connection username** to authenticate all the API calls coming from the connector linked to your connection.
```
